tsconfig.json

```diff
@@ -1,21 +1,18 @@
 {
     "compilerOptions": {
-      "strict": true,
-      "moduleResolution": "node",
-      "declaration": false,
-      "removeComments": false,
-      "alwaysStrict": true,
-      "pretty": false,
-      "module": "commonjs",
-      "target": "es2019",
-      "lib": [
-        "esnext",
-        "esnext.array",
-        "esnext.asynciterable",
-        "esnext.intl",
-        "esnext.symbol"
-      ],
-      "sourceMap": false,
-      "skipDefaultLibCheck": true
-    }
-  }
\ No newline at end of file
+        "target": "es2020",
```

Comment:
i'm not sure if this is different for ts,
but we for example use replaceAll = 21 feature.
So should this get changed?

Reply:
we can, but it does not matter as ts is only for typings

Reply:
ah, okay, then nvm I guess